recipes/pyagg/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.0.1" %}
+{% set sha256 = "30a1c5519696ff868104cfe7d33c8ec52e8698a8be6e21a3b6eb2d0c168f28aa" %}
+
+package:
+  name: pyagg
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/jwiggins/pyagg/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - numpy
+    - cython
+    - freetype
+  run:
+    - python
```

Comment:
Because of [this](https://github.com/jwiggins/pyagg/blob/master/setup.py#L125) you may need to specify the `numpy` dependency with: `numpy x.x`

That special syntax bakes the `numpy` version string into the package ensuring that the same `numpy` used at build time will be used at run time.